kubernetes/hpa.yaml

```diff
@@ -0,0 +1,17 @@
+kind: HorizontalPodAutoscaler
+apiVersion: autoscaling/v2beta1
+metadata:
+  name: torchserve
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: torchserve
+  # autoscale between 1 and 10 replicas
+  minReplicas: 1
+  maxReplicas: 5
+  metrics:
+  - type: External
+    external:
+      metricName: ts_queue_latency_microseconds
+      targetValue: "7000000m"
```

Comment:
@chauhang I'm from Walmart. I have some questions on the metrics used for autoscaling.

Is there a way to enable the autoscale based on multiple set of (**metrics**, **metrics query** and **targetValue**) combinations?
Say, if the metrics used are _ts_queue_latency_microseconds_, _ts_inference_requests_total_, _CPUUtilization_ each with it's own targetValue and metrics query. Can we set the autoscale work if **any**of these metrics conditions are met?

Reply:
https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-multiple-metrics